python_modules/libraries/dagster-dg/dagster_dg/cache.py

```diff
@@ -78,6 +77,7 @@ def clear_key(self, key: tuple[str, ...]) -> None:
 
     def clear_all(self) -> None:
         shutil.rmtree(self._root_path)
+        assert not self._root_path.exists()
```

Comment:
just confirming this was intentional and not a local testing thing, could imagine warning or something being less disruptive if this didn't happen for some obscure filesystem reason

Reply:
that actually was a testing thing, good catch